allvers/2000-02-29-GPL-2.0-from-gnu.org.html.notes

```diff
@@ -0,0 +1,5 @@
+Date: 2000-02-29 or earlier
```

Comment:
A more exact range is known: between April 24, 1999 and February 29, 2000

Reply:
Updated in 8a54a89bd97633bfb8a610c30dc10e6a62a19805